site/src/components/PageHeader.tsx

```diff
@@ -64,7 +65,15 @@ const PageHeader: React.SFC<PageHeaderProps> = ({
             </div>
           )}
           <div className={styles.mainSection}>
-            <h1 className={styles.headingText}>{headingText}</h1>
+            <div className={styles.headingTextContainer}>
```

Comment:
`headingTextContainer` should be able to replaced with the upcoming `Spacing` component in future